rllib/core/models/specs/checker.py

```diff
@@ -173,7 +173,7 @@ def check_input_specs(
     *,
     only_check_on_retry: bool = True,
     filter: bool = False,
-    cache: bool = False,
+    cache: bool = True,
```

Comment:
Wow, does that mean we never cached the checks in the RLModules?

Reply:
Never - by default - for the outputs, correct.